includes/core-changes/corefx/5.0/createcountersetinstance-throws-invalidoperation.md

```diff
@@ -0,0 +1,33 @@
+### CounterSet.CreateCounterSetInstance now throws InvalidOperationException if instance already exist
+
+Starting in .NET Core 5.0, <xref:System.Diagnostics.PerformanceData.CounterSet.CreateCounterSetInstance>(string `instanceName`) throws <xref:System.InvalidOperationException> instead of <xref:System.ArgumentException> if user attempt to create already existing instance.
```

Comment:
```suggestion
Starting in .NET 5.0, <xref:System.Diagnostics.PerformanceData.CounterSet.CreateCounterSetInstance(System.String)?displayProperty=nameWithType> throws an <xref:System.InvalidOperationException> instead of an <xref:System.ArgumentException> if the counter set already exists.
```